engine/src/main/resources/assets/atlas/gui.atlas

```diff
@@ -1,13 +1,12 @@
 {
-    "texture": "engine:gui",
-    "textureSize": [256, 256],
+    "texture": "engine:crosshair",
+    "textureSize": [220, 22],
     "grid": {
         "tileSize": [22, 22],
-        "gridDimensions": [9, 1],
-        "gridOffset": [24, 23],
+        "gridDimensions": [10, 1],
         "tileNames": [
             "crosshair", "crosshairCharge1", "crosshairCharge2", "crosshairCharge3", "crosshairCharge4",
-            "crosshairCharge5", "crosshairCharge6", "crosshairCharge7", "crosshairCharge8"
+            "crosshairCharge5", "crosshairCharge6", "crosshairCharge7", "crosshairCharge8", ""
```

Comment:
why we have empty tile?

Reply:
Because I made the texture file 220x22 with 10 tiles... I could also go for 198x22 and have only 9 tiles :shrug: